test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua

```diff
@@ -118,6 +115,7 @@ end
 commonFunctions:newTestCasesGroup("Postconditions")
 
 function Test.Postcondition_Restore_preloaded_pt()
+  commonPreconditions:RestoreFile("smartDeviceLink.ini")
```

Comment:
@istoimenova, Please rename "Test.Postcondition_Restore_preloaded_pt" to "Test.Postcondition_Restore_Files"